### PR TITLE
Fix originate and compile commands in command pallet

### PR DESCRIPTION
**Pull request by AlirezaHaghshenas**
# 🏗️ PR ➾

Fixes # (issue)

## 🪂 Pre-Merge Checklist

- [ ] 🐬 I have commented my code, particularly in hard-to-understand areas
- [ ] 🦀 Automated tests have been written and added to this PR
- [ ] 🏖️ New and existing unit tests pass locally and in CI
- [ ] 🤿 Corresponding changes have been made to all documentation
- [ ] 🔱 The test plan has been implemented and verified by an SDET
- [ ] 🏄‍♂️ Myself and others have built this branch and done manual testing on the change
- [ ] ⛵ A summary describing the change has been written for release notes

----------------------------------------------------------------------------------------------------------------------------

## 🪁 Description

Please include a summary of the change and add the issue number this is related to above. Please also include relevant motivation and context for the change
(e.g. what was the existing behavior, why did it break, etc.)

## 🎢 Test Plan

Please describe the testing strategy for this change

----------------------------------------------------------------------------------------------------------------------------

### 🛸 Type of Change

- [ ] 🐟 Minor change (non-breaking change of very limited scope)
- [ ] 🦑 Bug fix (non-breaking change which fixes an issue)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🌊 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🐳 Major refactor (breaking changes and significant impact to the codebase)

----------------------------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Review by hu3man — APPROVED** [review]
This works great, nice work @AlirezaHaghshenas!

File: taqueria-vscode-extension/src/lib/gui/ContractsDataProvider.ts [hunk 1]
```diff
@@ -23,7 +23,8 @@ export class ContractsDataProvider extends TaqueriaDataProviderBase
 			new ContractTreeItem(
 				path.basename(uri.path),
 				vscode.TreeItemCollapsibleState.None,
-				extensions[path.extname(uri.path).toLowerCase()],
+				smartContractLanguages.find(info => info.fileExtensions.indexOf(path.extname(uri.path).toLowerCase()) !== -1)
```
Comment (jchenche) [inline taqueria-vscode-extension/src/lib/gui/ContractsDataProvider.ts:26]: What does this line do?

File: taqueria-vscode-extension/src/lib/helpers.ts [hunk 2]
```diff
@@ -34,15 +35,61 @@ export const COMMAND_PREFIX = 'taqueria.';
...
+		pluginName: '@taqueria/plugin-ligo',
+		compilerName: 'ligo',
+	},
+	{
+		fileExtensions: ['.py'],
+		pluginName: '@taqueria/plugin-smartpy',
+		compilerName: 'smartpy',
+	},
+	{
+		fileExtensions: ['.ts'],
```
Comment (jchenche) [inline taqueria-vscode-extension/src/lib/helpers.ts:82]: We don't support Typescript in SmartPy yet btw. Only Python

File: taqueria-vscode-extension/src/lib/gui/ContractsDataProvider.ts (hunk 1 above)
Comment (AlirezaHaghshenas) [reply taqueria-vscode-extension/src/lib/gui/ContractsDataProvider.ts:26]: This finds the `SmartContractLanguageInfo` that matches the file extension of the current contract, and sets the language name to be shown in the TreeViewItem

File: taqueria-vscode-extension/src/lib/helpers.ts (hunk 2 above)
Comment (AlirezaHaghshenas) [reply taqueria-vscode-extension/src/lib/helpers.ts:82]: @hu3man So should I not enable it when the user has a `.ts` file open? For instance?

Comment (hu3man) [reply taqueria-vscode-extension/src/lib/helpers.ts:82]: Yes, that would be ideal behaviour! 

May want to leave a comment there that we may need to refactor if and when we support TS via SP

File: taqueria-vscode-extension/src/lib/gui/ContractsDataProvider.ts (hunk 1 above)
Comment (AlirezaHaghshenas) [reply taqueria-vscode-extension/src/lib/gui/ContractsDataProvider.ts:26]: I think it's better to add a few helper functions to simplify these.

**Review by jchenche — APPROVED** [review]

**Review by sinapsist — APPROVED** [review]
LGTM

**Review by hu3man — APPROVED** [review]
🚀

**Review by jchenche — APPROVED** [review]

**Pull request merged**